qiskit_nature/properties/second_quantization/driver_metadata.py

```diff
@@ -37,6 +38,14 @@ def __init__(self, program: str, version: str, config: str) -> None:
         self.program = program
         self.version = version
         self.config = config
+        warn_deprecated(
+            "0.5.0",
+            old_type=DeprecatedType.CLASS,
+            old_name="qiskit_nature.properties.second_quantization.DriverMetadata",
+            new_type=DeprecatedType.CLASS,
+            new_name="qiskit_nature.second_q.problems.DriverMetadata",
```

Comment:
This class will likely also be removed in its entirety.
A replacement is not yet planned as this information is not relevant to the stack itself, thus, is not going to be propagated down in the new workflow (until it becomes necessary per a specific feature request). Again, we could link to #705